circuit/src/threshold/native.rs

```diff
@@ -1,10 +1,19 @@
+use std::str::FromStr;
+
 use crate::{
 	params::rns::{compose_big_decimal_f, decompose_big_decimal},
+	utils::big_to_fe,
 	FieldExt,
 };
 use halo2::{arithmetic::Field, halo2curves::bn256::Fr};
+use num_bigint::BigUint;
 use num_rational::BigRational;
 
+/// Number of limbs for threshold check
+pub const NUM_LIMBS: usize = 55;
+/// Power of 10 for threshold check
+pub const POWER_OF_TEN: usize = 76;
+
```

Comment:
What does this corresponds to?
4 neighbours or 128 neighbours?

Reply:
Ah, now I realise you tried to find the constants for both of them.
You should look for different constants for each of them.

Reply:
Oh, so you mean that we don't want the universal constants. 
But, we need to calculate the constants for every case. (atm, 4 peers & 128 peers)
Right?

Reply:
Yes